src/commands/scanner/scannerCommand.ts

```diff
@@ -43,7 +44,7 @@ export abstract class ScannerCommand extends SfdxCommand {
   }
 
   protected displayError(msg: string): void {
-    this.ux.error(msg);
+    throw new SfdxError(msg);
```

Comment:
What motivated this change? As a result of this change, this method now behaves differently inside or outside a try-catch block. I'm concerned that there could be unforeseen consequences to this.

Reply:
I was looking for a way to halt the process right here. But, like I mentioned in scrum, the error doesn't look nice since this is thrown in Child process' context instead of Sfdx. I'm changing this to display error and do a `process.exit(errorCode)`. Will push updated code soon.